### ISSUE TITLE
⌨ Career planning: Lacks change in context (page title, h1, and breadcrumbs)

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#12883]
## ⌨ Accessibility Issue

When navigating to "edit career planning" from the dashboard, the page title, headings, and breadcrumbs do not change, failing to indicate a change in context. This can disorient users, as the new section appears to be the same as the dashboard, even though the content has changed.

### WCAG 2.1 Criteria

- WCAG 2.4.2 Page Titled – Web pages should have titles that describe their topic or purpose.
- Heuristic: Visibility of System Status – The system should always keep users informed about what is going on, including their location within the site.
- Heuristic: Consistency and Standards – Consistent use of titles, headings, and breadcrumbs helps users understand navigation patterns.

## 🦋 Expected Behaviour

When the user navigates to "edit career planning," the page title, headings, and breadcrumbs should update to reflect the new section, clearly indicating that the user has moved to a different area of the application. This aligns with how the "personal information" section is displayed, which properly orients the user to the new context.

## 📋 Steps to Reproduce

1. Navigate to applicant dashboard by logging into the platform
2. Make sure the role is set to "Applicant"
3. Scroll to "Career development" section. Click "edit career planning"

## 📸 Screenshot

![Image](https://github.com/user-attachments/assets/ad520eaf-d9ee-4b15-857e-b6d1ec471252)

## 🙋‍♀️ Proposed Solution

Change the page title, header and breadcrumbs to reflect the change in context - similar to the "personal information" page.

**Comment by esizer** [comment]
Just want to try and clear stuff up here to get a sense on how we want to address this.

So the page title is correct, the issue may be that when you click "edit career planning" you are actually going to a sub-sectioin of this page. However, since right now there is only one section on the page it makes it seem like a bug. Eventually we will be adding more sections.

Is is possibly just confusing that currently there is oinly a single section so it seems like the page is actually "Career planning" and not the employee profile? 🤔

**Comment by MaddyDaigle** [comment]
Hmm... but if the user selects "career planning" from the "Your information" section, the headers, titles, etc update as expected.

If accessed through "Your information" section:

![Image](https://github.com/user-attachments/assets/597d53b5-ae9d-4498-a97a-d8beb8d6e6c1)

If accessed through "career development" > "Edit career planning" (what you see above in the ticket).

**Comment by MaddyDaigle** [comment]
Possibly... when when you click "Personal Information" Under the "Your Information" section; the header, title and breadcrumbs all change. Same with "career experience".  It should be consistent.

**Comment by esizer** [comment]
> [@esizer](https://github.com/esizer) Right... it took me a bit to understand where I was in the context. Since there's "career development", "GC employee profile" and "career planning". I think I was confused by the different wording and functionality and where they all fit. Also, because we used to refer to the "your information" as "filling out your profile" on the talent platform; I was a bit confused between that and the "GC employee profile". It might be labelling and wording confusion.

Okay, I agree. The interaction is a little confusing and is exacerbated when using ATs. I'll mull over some solutions if we are unable to line up the copy so that it matches.
